examples/Daily-temperature-example.ipynb

```diff
@@ -0,0 +1,1020 @@
wOBOcsInVZBUuyp+Hnmu6ma4EqsabT8RJQOA///MUDsy88fM3JOZeyYlOftAV0ssf3kyV0IGo62C9JLBUOEDf+7yzq7KGeEkTheHPiE3jnBJ12QMPSNZ2UskGnGDSywUVhX4LwBu9X++FcAkMc0Ry8R7+uC6ni3wyS09se7ZixyPlHBLZDShfPWhBg6UfHJvl5eP6jwpGrHcryjfjhEiulCI6FsA/QA0JKIMAE8DeAXABCIaDmAngGtlNtIqXZrVrTQ77yQq3BbBICJ0aVYX/x7WCUu3H8SDg9qpblLUUDFCAlCfE0QjlpqK8u0YIaICZ+YbQnzlnqS4LsUd6vukQhl+3mmO1uuLBZ68uCOmrNtXbpu2wDVOEbUrMd2AW1aXa3Uij5YNauCX+8pPsLnZZ6oxj5QkbILQClwibnGh1InhpcZO0K5CJkqtv6OLwZ0ao3/7JNSploAPb3JXeGbUJbNyEyr0d/XEeBwvKjnx94WdGldSMBq5xGr0TbRCRBh3ewqKS0qF19y0i7taE2XUk1wFJxhfDE9BYjyhemI8frnvXHxyS0/H2xBruGSgpZGM25Q3oC1wqbxwRRcMGDMHJaXs2NDr7Fb1sXDUQFRJiFObpS2GrNCE+Ni5Vo270ApcIqc2qIkFTwxAzvEitHcwf7PIkmWWiSGzNDE+DmOuPRNfLt6BO3SUj8ZBtAKXTJO61dBEQgEJjbu4+qzmuPqs5qqboYkx3OfU0UQHMeRC0WhUoRW4RqPReBStwDUajcajaAWukUKCXs2i0UhHK3CNMMryrNStnogruskvLKzRxDo6CkUjjFFDO6BP6wbo3LRuTNcp1GicQitwjTAS4+MwsGP0VILRaNyOdqFoNBqNR9EKXKPRaDyKVuAajUbjUbQC12g0Go+iFbhGo9F4FK3ANRqNxqNoBa7RaDQehZys20hEWQB2WDy8IYADApvjBdmxJlelbH3NsSHbq9d8KjMnVdzoqAK3AxGlMrOS+mCqZMeaXJWy9TXHhuxou2btQtFoNBqPohW4RqPReBQvKfCPY1B2rMlVKVtfc2zIjqpr9owPXKPRaDTl8ZIFrtFoNJpAmFnZPwCfAcgEsDZg25kAFgH4C8CvAOr4tycCGO/fngZgdLjzyJYLoAWAWf5t6wA84KDsagCWAljtl/2sU/fa/308gJUAfnPwN073b18FINXh/lUPwEQAG/zfnePAb9zef61l/44AeNDBa37I37fWAvgWQDWH5D7gl7nOyPVakF0FwDj/9tUA+gUcc5Z/+xYA78DvoXBA7osAdgHINXK95dpg9gCR/wD0BdCjwg1YBuAC/+c7ADzv/3wjgO/8n2vA90C3CnUe2XIBJAPo4d9eG8AmAJ0ckk0AagU8DEsA9HbiXvu3PQzgGxhT4KJ+43QADRX1r/EA7uSTD2I9p+61f3s8gH3wxQI70b+aAdgOoLr/uwkAbnNAbhf4lHcN+GoVTAfQVvA13wtgnP9zIwDLAcT5/14K4Bz4nq8/AAx1SG5v+PSJaQWu1IXCzHMBHKqwuT2Auf7P0wBcXbY7gJpElACgOoBC+KySUOeRKpeZ9zLzCv/5jsJnSUSsIyZINjNzrn+fRP+/sJMZou41ETUHcAmAsZGuVaRcK4iQTUR14HtQP/Wfs5CZD8uWW+HYgQC2MnPERXACZScAqO7/rgaAPQ7I7QhgMTPnMXMxgDkArhR8zZ0AzPAflwngMICeRJQMn7W8iH1a9QsAV8iW6/97MTPvjXSdwXCjD3wtgMv8n6+Fz1UB+IawxwDsBbATwBvMbFhpy5RLRK0AdIfPEnZENhHFE9Eq+IZw05jZimwr1/w2gMcBlFqQZ0cuA/iTiJYT0QgHZZ8OIAvAOCJaSURjiaimA3IDuR4+N4ZVTMlm5t0A3vBv2wsgh5n/lC3Xv39fImpARDUAXBxwjCjZqwFcTkQJRHQafG6TFvAZXxkBx2fAgEEmQK4t3KjA7wBwLxEth881UejfngKgBEBTAKcBeISITlctl4hqAfgRPn+dVWvRtGxmLmHmbgCaA0ghoi6y5RLRMACZzLzcykValev/7lxm7gFgqP/Yvg7JToBvmPwBM3eHT/GMckAuAICIqsCnEH6wINOSbCI6BcDl/m1N4bOWb5Itl5nTALwKn+U6BT6lV2xBbjjZn8GnnFPhM0YW+mVQkHNYCdEzK9cWrquJycwbAAwGACJqB99wHfD5zaYwcxGATCJaAN8QZJsquUSUCJ/y/pqZf3JSdsCxh4loNoAh8L39ZcrtDuAyIroYvonUOkT0FTOberitXC8z7/Efm0lEP8OnBOZWOrl42XMBZASMcCbCggK38RsPBbCCmfeblWlDNgPYzsxZ/mN+AtAHwFeS5W5j5k/hd1cR0UsobxXblu13zTxUth8RLQSwGUA2fMZQGc0RwW0kSK4tXGeBE1Ej//9xAP4F4EP/VzsBDCAfNeFz/G9QJZeICL6OlsbMbzosO4mI6vmPqQ5gECzcC7NymXk0Mzdn5lbwDetnmlXeFq+3JhHV9h9TE74HxNTLysY17wOwi4ja+/cbCGC9bLkBh94Ae+4TK7J3AuhNRDX8/XwgfHM8suUGHtMSwFWweO2hZPuvqab/84UAipl5vd8HfZSIevuv+RYAk2TLtXJt5TA76ynyH3w/zl4ARfC9aYfDF0a0yf/vFZxcbFQLvmHkOvgeoMfCnUe2XADnwWeprMHJUK+LnbhmAF3hC+NbA58i+49T9zrgfP1gLApFxPWeDt9wuixs8imH+1c3+Ia+awD8D8ApDsmtAeAggLoKnqln4VOqawF8CaCqQ3Ln+betBjBQwjW3ArARvhfSdARE9sA3ClgLYCuA98qOcUDua/7jS/3/P2P099YrMTUajcajuM6FotFoNBpjaAWu0Wg0HkUrcI1Go/EoWoFrNBqNR9EKXKPRaDyKVuAajUbjUbQC12g0Go+iFbhGo9F4lP8HiDIGj23JcSIAAAAASUVORK5CYII=\n",
```

Comment:
I hope the pics won't take too much space on github :)

Reply:
Yeah... I wonder how we should treat notebooks in general, it's kind of a pain to have them tracked by git (as usually executing them changes them).